backend/pkg/api/panel.go

```diff
@@ -152,7 +152,7 @@ func (panel *TablePanel) GetData(authConfig auth.AuthConfig) error {
 		return err
 	}
 
-	url := authConfig.AuthURL() + "/api/tsdb/query"
+	url := authConfig.AuthURL() + "/api/ds/query"
```

Comment:
yes, the URL has changed. Should really be using the backend query helpers I guess.

Reply:
:(

if I remember the sdk didn't provide anyway to do this... quick look at activity in the sdk repo and it seems they might added this functionality with https://github.com/grafana/grafana-plugin-sdk-go/pull/328 

but not sure..

😢